pgcli/main.py

```diff
@@ -440,11 +461,22 @@ def connect(self, database='', host='', user='', port='', passwd='',
             try:
                 pgexecute = PGExecute(database, user, passwd, host, port, dsn,
                                       application_name='pgcli', **kwargs)
-                if passwd:
+                if passwd and keyring and self.config["main"].get("keyring", True):
```

Comment:
I would add the default `keyring = True` to `pgclirc`, for better discoverability, and I'd also put in a comment explaining what it is. People tend to rely on `pgclirc` to tell them what `pgcli` can do.

Reply:
Ah yes, thanks for reminding. I forgot to change pgclirc